python/paddle/fluid/dygraph/jit.py

```diff
@@ -132,7 +132,10 @@ def _declarative_(dygraph_func):
     """
     Converts imperative dygraph APIs into declarative function APIs. Decorator
     @declarative handles the Program and Executor of static mode and returns
-    the result as a dygraph VarBase.
+    the result as a dygraph VarBase. Users could use the returned dygraph
+    Varbase to do imperative training, inference, or other operations. If the
```

Comment:
1. The concept of `VarBase` is not exposed to users. Better to use `Tensor ` instead.
2. The returned result may contain more than one Tensor.

Reply:
Done